src/main/java/frc/robot/util/Blinkin.java

```diff
@@ -90,6 +91,9 @@ public static void colorPeriodic() {
     if (isDualColor) {
       dualColorPeriodic();
     } else {
+      m_timer.reset();
+      m_timer.stop();
+      dualColorIndex = 0;
```

Comment:
Im not sure if this is necessary, it's fine to keep the timer "running" because really it's just checking a clock